### PR TITLE
Fix/etp 111 hook not passing ticket

### PR DESCRIPTION
**Pull request by redscar**
### 🎫 Ticket

[ETP-111]

### 🗒️ Description

This pull request coincides with one for ET+ (https://github.com/the-events-calendar/event-tickets-plus/pull/1226).

Updated the tribe_events_tickets_metabox_edit_ajax_advanced to have three parameters. 

New Feature

To test add this code to your functions.php
```
add_action( 'tribe_events_tickets_metabox_edit_ajax_advanced', static function(){
    $args = func_get_args();
    var_dump($args);
},15,3 );
```

### 🎥 Artifacts 

image
Screenshot displays the var_dump from the above code snippet. The ticket ID is 15.

### ✔️ Checklist
- [x] I've included a changelog entry both in readme.txt and changelog.txt files. 
- [x] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ETP-111]: https://theeventscalendar.atlassian.net/browse/ETP-111

### CODE REVIEW

File: src/Tribe/Commerce/PayPal/Main.php [hunk 1]
```diff
@@ -1804,8 +1804,9 @@ public function do_metabox_advanced_options( $post_id, $ticket_id ) {
 		 *
 		 * @param int Post ID
 		 * @param string the provider class name
+		 * @param int the  id of the ticket
```
Comment (sc0ttkclark) [inline src/Tribe/Commerce/PayPal/Main.php:1807]:
```suggestion
		 * @param int $ticket_id The ticket ID.
```

File: readme.txt [hunk 2]
```diff
@@ -182,7 +182,7 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 = [5.1.6] TBD =
 
-
+* Tweak - Added $ticket_id to 'tribe_events_tickets_metabox_edit_ajax_advanced'. [ETP-111]
```
Comment (juanfra) [inline readme.txt:185]:
```suggestion
* Tweak - Added `$ticket_id` parameter to the `tribe_events_tickets_metabox_edit_ajax_advanced` filter. [ETP-111]
```
Maybe we can add some more info of the change. We normally tend to use backticks when using variable names and hooks in the changelog entries, as they normally are use in different places to represent code format.

File: changelog.txt [hunk 3]
```diff
@@ -1,5 +1,7 @@
 == Changelog ==
 
+* Tweak - Added $ticket_id to 'tribe_events_tickets_metabox_edit_ajax_advanced'. [ETP-111]
```
Comment (juanfra) [inline changelog.txt:3]:
```suggestion
= [5.1.6] TBD =

* Tweak - Added `$ticket_id` parameter to the `tribe_events_tickets_metabox_edit_ajax_advanced` filter. [ETP-111]
```
Seems like the `changelog.txt` is out of synch, good opportunity to set the upcoming version there as well. Not part of the fix, but I normally prefer to fix them as soon as they see them so we can improve things as we get going.

**Review by juanfra — COMMENTED** [review]
⚡Looking good! a couple notes about the readme entry, to align with what we normally do.

File: src/Tribe/Commerce/PayPal/Main.php [hunk 4]
```diff
@@ -1804,8 +1804,9 @@ public function do_metabox_advanced_options( $post_id, $ticket_id ) {
 		 *
 		 * @param int Post ID
 		 * @param string the provider class name
+		 * @param int $ticket_id The ticket ID.
 		 */
-		do_action( 'tribe_events_tickets_metabox_edit_ajax_advanced', $post_id, $provider );
+		do_action( 'tribe_events_tickets_metabox_edit_ajax_advanced', $post_id, $provider, $ticket_id  );
```
Comment (juanfra) [inline src/Tribe/Commerce/PayPal/Main.php:1809]:
```suggestion
		do_action( 'tribe_events_tickets_metabox_edit_ajax_advanced', $post_id, $provider, $ticket_id );
```

**Review by bordoni — DISMISSED** [review]

**Pull request merged**